gamestonk_terminal/options/syncretism_view.py

```diff
@@ -297,15 +300,11 @@ def check_presets(preset_dict: dict):
                 error += f"{key} : {value},  Should be [true/false]\n"
 
         elif key == "tickers":
-            if value[0] != '"':
-                error += f'{key} : {value}, ticker should have quotes "AAPL" \n'
-            else:
+            for ticker in value.split(","):
                 try:
-                    for ticker in value.split(","):
-                        if ticker[0] != '"':
-                            error += f'{key} : {ticker}, all tickers should have quotes "AAPL","GME" \n'
-                except Exception:
-                    error += f"Error splitting tickers: {value} \n"
+                    eval(ticker)
```

Comment:
Can we check if the stock exists from yfinance? Or does the API returns this?

Reply:
One could, but honestly this is likely going to be blank since you want to find different stocks with your criteria, so it seems like overkill to have to check.  If the ticker doesn't exist then it returns empty, not an error

Reply:
So, my opinion is that if the user provides a ticker that doesn't exist, we want to warn them about that.

It isn't overkill because if the user doesn't provide any ticker (which will be the case almost always, as you mentioned) then we don't use yfinance to check on ticker